contrib/devtools/README.md

```diff
@@ -119,7 +119,25 @@ Configuring the github-merge tool for the bitcoin repository is done in the foll
 
     git config githubmerge.repository bitcoin/bitcoin
     git config githubmerge.testcmd "make -j4 check" (adapt to whatever you want to use for testing)
-    git config --global user.signingkey mykeyid (if you want to GPG sign)
+    git config --global user.signingkey mykeyid
```

Comment:
Removed the `(if you want to GPG sign)` intentionally here. I know it's unrelated to this patch, but GPG signing has been mandatory for forever and I don't think this one-line doc change warrants another commit.